WanderLost/WanderLost/Server/Controllers/MerchantHub.cs

```diff
@@ -98,8 +98,6 @@ public async Task UpdateMerchant(string server, ActiveMerchant merchant)
         merchant.UploadedBy = clientIp;
         merchant.UploadedByUserId = Context.UserIdentifier;
         merchant.RequiresProcessing = true;
-        //Add an auto-upvote so the user can see their own submissions by default
-        merchant.ClientVotes.Add(new Vote() { ClientId = clientIp, UserId = Context.UserIdentifier, VoteType = VoteType.Upvote });
```

Comment:
Was there a reason this was removed? Seems like an accidental change because it regresses some functionality.

Reply:
If this is a requested feature and is local-only - then sure, I agree that it should be reverted

Reply:
Ok, I'll probably just re-base that commit away and then merge this in to main later today, probably after work.